mail/dmarc-report-converter/Portfile

```
@@ -0,0 +1,64 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/tierpod/dmarc-report-converter v0.6.5
+categories          mail
+maintainers         {gmail.com:davidgilman1 @dgilman} openmaintainer
+license             MIT
+
+description         Convert dmarc reports from xml to human-readable formats
+long_description    ${description} Files can be located on a local filesystem or on an IMAP server.
```

When reusing the value of `${description}` elsewhere, such as in `long_description`, you need to use Tcl's expansion operator (`{*}`):

```suggestion
long_description    {*}${description} Files can be located on a local filesystem or on an IMAP server.
```